lib/components/Editor/CustomExtensions/Markdown/useMarkdownEditor.js

```diff
@@ -47,11 +47,11 @@ const useMarkdownEditor = ({ content, onUpdate, onSubmit }) => {
     (e) => {
       if (e.key !== "Enter" || !(e.metaKey || e.ctrlKey)) return;
       inputFieldRef.current?.blur();
-      if (!onSubmit) return;
+      if (!markdownMode || !onSubmit) return;
```

Comment:
looks like it's better to move this to top of this function like
```
if(!markdownMode) return
```
early return if not markdown mode so that other statements need not be evaluated